pkg/service/auth.go

```diff
@@ -83,7 +83,8 @@ func (m *APIKeyAuthMiddleware) ServeHTTP(w http.ResponseWriter, r *http.Request,
 }
 
 func GetGrants(ctx context.Context) *auth.ClaimGrants {
-	claims, ok := ctx.Value(grantsKey{}).(*auth.ClaimGrants)
```

Comment:
this line was causing compile to fail with Go 1.18. It now compiles! 👨 🥣 🦀 @cnderrauber

Reply:
Great!